SparkPost/2web2ui
# FE-1080 - For Hibana only, reposition Snackbar global alerts to the bottom left of the viewport

[FE-1080](https://jira.int.messagesystems.com/browse/FE-1080)

### What Changed
- Refactored `Animator` component from class to functional in order to leverage the `useHibanaOverride` hook
- Added new styles to reposition global alerts

### How To Test
- Verify global alerts render in the right spot according to the a/c
- Verify they continue to render in the top right in the OG theme

### To Do
